gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/formatter/model/IStepContainer.java

```diff
@@ -0,0 +1,25 @@
+package gherkin.utils.formatter.model;
+
+import java.util.List;
+
+import static io.cucumber.messages.Messages.GherkinDocument.Feature;
+
+public interface IStepContainer {
```

Comment:
Please use `StepContainer`. It is a proper domain object so there is no need to prefix it with an indication that it is an interface.

Reply:
Thank you, I have already fixed this class name.